src/system-health/health_checker/config.py

```diff
@@ -0,0 +1,140 @@
+import os
+import json
+from sonic_py_common import device_info
+
+
+class Config(object):
+    """
+    Manage configuration of system health.
+    """
+
+    # Default system health check interval
+    DEFAULT_INTERVAL = 60
+
+    # Default boot up timeout. When reboot system, system health will wait a few seconds before starting to work.
+    DEFAULT_BOOTUP_TIMEOUT = 300
+
+    # Default LED configuration. Different platform has different LED capability. This configuration allow vendor to
+    # override the default behavior.
+    DEFAULT_LED_CONFIG = {
+        'fault': 'red',
+        'normal': 'green',
+        'booting': 'orange_blink'
+    }
+
+    # System health configuration file name
+    CONFIG_FILE = 'system_health_monitoring_config.json'
+
+    # Monit service configuration file path
+    MONIT_CONFIG_FILE = '/etc/monit/monitrc'
+
+    # Monit service start delay configuration entry
+    MONIT_START_DELAY_CONFIG = 'with start delay'
+
+    def __init__(self):
+        """
+        Constructor. Initialize all configuration entry to default value in case there is no configuration file.
+        """
+        self.platform_name = device_info.get_platform()
+        self._config_file = os.path.join('/usr/share/sonic/device/', self.platform_name, Config.CONFIG_FILE)
+        self._last_mtime = None
+        self.config_data = None
+        self.interval = Config.DEFAULT_INTERVAL
+        self.ignore_services = None
+        self.ignore_devices = None
+        self.user_define_checkers = None

```

Comment:
Please rename `user_defined_checkers`, and update all references throughout all files.